src/HttpsUri.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-http for the canonical source repository
+ * @copyright Copyright (c) 2005-2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-http/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Http;
+
+use Zend\Uri\Http as Http;
+
+class HttpsUri extends Http
```

Comment:
There is no reason to introduce this class.
`$uri->setScheme('https');` provides exactly the same behavior